code/modules/reagents/reagents/toxin.dm

```diff
@@ -437,6 +437,20 @@
 	meltprob = 30
 	taste_multi = 1.5
 
+/datum/reagent/toxin/acid/xeno_acid
+	name = "Xenomorph acid"
+	description = "Neon green, lethal and utterly corrosive. Do not ingest."
+	color = "#8E18A9" // rgb: 142, 24, 169
+	toxpwr = 1.5
+	meltprob = 30
+	taste_multi = 1.5
+	custom_metabolism = REAGENTS_METABOLISM
+
+/datum/reagent/toxin/acid/on_mob_life(mob/living/L, metabolism)
+	var/acid_damage = 2 * toxpwr * REM
+	L.take_limb_damage(acid_damage, acid_damage) //REM being 0.5, so 3 damage per tick, 15 damage per second
```

Comment:
on mob life ticks once every 2 seconds though

Reply:
I think he meant 1.5, not 15.

Reply:
Oh that's an old comment from a different damage formulation.